docs/conf.py

```diff
@@ -121,6 +122,34 @@ def check_dot_command():
 # The full version, including alpha/beta/rc tags.
 release = version
 
+
+iso_gpg_key: str
+oss_gpg_key: str
+
+try:
+    # if product's conf.py injected an iso_gpg_key, use that one
+    iso_gpg_key
+except NameError:
+    # else set a dummy value
+    iso_gpg_key = "<gpg_key>"
+
+try:
+    # if product's conf.py injected oss_gpg_key, use that one
+    oss_gpg_key
+except NameError:
+    # else set a dummy value
+    oss_gpg_key = "<gpg_key>"
+
+
+version_major = int(version.split(".")[0])
+rst_prolog = f"""
+    .. |iso_major| replace:: {version_major}
```

Comment:
Minor preference to keep it generic here as well for consistency.
```suggestion
    .. |version_major| replace:: {version_major}
```